app/src/main/res/layout/activity_signup.xml

```diff
@@ -1,14 +1,14 @@
 <FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Please revert the changes to activity_signup.xml. They look like they are just formatting changes.